docs/flow/modin/experimental/xgboost.rst

```diff
@@ -63,11 +63,28 @@ for any own case.
 3. ``ray.util.placement_group`` is creating to block of all available Ray resources. After that objects of ``ModinXGBoostActor``
```

Comment:
"block of all" - probably better use "reserve all" "occupy all"

Reply:
fixed